packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs

```diff
@@ -42,13 +48,37 @@ impl<C> Platform<C> {
             ));
         }
 
+        tracing::info!(
+            "{}",
+            format!(
+                "[ody] check_for_desired_protocol_upgrade: \
+        total_hpmns:{}\
+        required_upgraded_hpmns:{}\
+        current_votes:{}\
+        versions_passing_threshold:{}",
+                total_hpmns,
+                required_upgraded_hpmns,
+                votes_str,
+                versions_passing_threshold.len(),
+            )
+        );
+
         if !versions_passing_threshold.is_empty() {
             // same as equals 1
             let next_version = versions_passing_threshold.remove(0);
 
             // TODO: We stored next version here previously.
             //  It was never used so we can temporary remove it from here and move it to Epoch trees in upcoming PR
 
+                tracing::info!(
```

Comment:
I would move it there https://github.com/dashpay/platform/blob/a12abe641322e503b1cce4195b34783750a77938/packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/upgrade_protocol_version/v0/mod.rs#L68-L67